src/plugins/controls/public/options_list/components/options_list_strings.ts

```diff
@@ -28,9 +28,54 @@ export const OptionsListStrings = {
       }),
   },
   editor: {
-    getAllowMultiselectTitle: () =>
-      i18n.translate('controls.optionsList.editor.allowMultiselectTitle', {
-        defaultMessage: 'Allow multiple selections in dropdown',
+    getSelectionOptionsTitle: () =>
+      i18n.translate('controls.optionsList.editor.selectionOptionsTitle', {
+        defaultMessage: 'Selections',
+      }),
+    selectionTypes: {
+      multi: {
+        getLabel: () =>
+          i18n.translate('controls.optionsList.editor.multiSelectLabel', {
+            defaultMessage: 'Allow  multiple selections',
+          }),
+      },
+      single: {
+        getLabel: () =>
+          i18n.translate('controls.optionsList.editor.singleSelectLabel', {
+            defaultMessage: 'Only allow a single selection',
+          }),
+      },
+    },
+    getSearchOptionsTitle: () =>
+      i18n.translate('controls.optionsList.editor.searchOptionsTitle', {
+        defaultMessage: `Searching`,
+      }),
+    searchTypes: {
+      prefix: {
+        getLabel: () =>
+          i18n.translate('controls.optionsList.editor.prefixSearchLabel', {
+            defaultMessage: 'Prefix',
+          }),
+        getTooltip: () =>
+          i18n.translate('controls.optionsList.editor.prefixSearchTooltip', {
+            defaultMessage: 'Matches values that begin with the given search string.',
+          }),
+      },
+      wildcard: {
+        getLabel: () =>
+          i18n.translate('controls.optionsList.editor.wildcardSearchLabel', {
+            defaultMessage: 'Contains',
+          }),
+        getTooltip: () =>
+          i18n.translate('controls.optionsList.editor.wildcardSearchTooltip', {
+            defaultMessage:
+              'Matches values that contain the given search string. Results might take longer to populate.',
+          }),
+      },
+    },
+    getAdditionalSettingsTitle: () =>
+      i18n.translate('controls.optionsList.editor.additionalSettingsTitle', {
+        defaultMessage: `Additional settings`,
```

Comment:
Copy reference: 

![Jun-01-2023 14-50-27](https://github.com/elastic/kibana/assets/8698078/0b7db08c-0322-450a-a0fa-cb7b9f08afaa)

@elastic/ux-writers

Reply:
The new fields here LGTM!

My only comment here would be on the already existing "Ignore timeout for results" label. The 1st sentence of its tooltip sounds much clearer to me even if it's longer. Please ignore if my comment is not relevant or out of scope here :)

Reply:
This is one of the settings that I think works well when it sounds a little technical. If you select it, it's actually `ignoring the timeout` behind the scenes which can make it take _way longer_.

I worry that if the option is friendly sounding like `Wait to display results until the list is complete` people will select it when they don't need it.

That said, I think there is room for improvement in the copy.

Reply:
> I worry that if the option is friendly sounding like Wait to display results until the list is complete people will select it when they don't need it.

That's a very good point - this should only be used when it is **absolutely necessary**, so making it sound a little more technical and "scary" is probably the right call here 😆